tapdance/logger.go

```diff
@@ -17,13 +17,16 @@ func (f *formatter) Format(entry *logrus.Entry) ([]byte, error) {
 var logrusLogger *logrus.Logger
 var initLoggerOnce sync.Once
 
+const build string = "[BUILD]"
```

Comment:
are we planning to use this outside of `Logger()` singleton? if not, would be nicer to move `build` variable into the function